components/kyma-metrics-collector/Makefile

```diff
@@ -34,3 +34,6 @@ go-mod-check-local:
 
 test-alerts:
 	promtool test rules ${PROMETHEUSRULES_PATH}/alerts_test.yaml
+
+update_metrics_docs:
```

Comment:
Where will this make target be triggered? Or is this the same setup as in kyma repository where governance job fails?

Reply:
atm there is no such thing... similar to eventing-controllers metric documentation generation... 
but that is a good point. we should introduce it